notes

```diff
@@ -1,2 +1,13 @@
 ;; -*- mode: outline; outline-regexp: " *[-\+]"; indent-tabs-mode: nil -*-
 
+- installing npm and babel
+ - ./node_modules/.bin/babel -o view/pages/assets/js/wab.js view/js/wab.js view/js/list.js view/js/obj.js view/js/view.js"
```

Comment:
consider using `--out-dir` or `-d` and `-w` along with `--presets es2015` when we make the switch to ES6 modules

Reply:
`-o` and --out-dir` are the same. `--minify` is something we should add as well once we get the new code debugged.

Reply:
they're not the same.. please check `babel --help` once more:
```
    -o, --out-file [out]                 Compile all input files into a single file
    -d, --out-dir [out]                  Compile an input directory of modules into an output directory
```

Reply:
But **currently** in our case, `-o` is the right choice. Once we make a proper move to ES6 modules, we can switch to the directory option

Reply:
You are correct although `-o` or `--out-file` is what we want.